shell/src/main/java/alluxio/cli/validation/NativeLibValidationTask.java

```diff
@@ -0,0 +1,58 @@
+package alluxio.cli.validation;
+
+import alluxio.cli.ValidateUtils;
+import alluxio.conf.AlluxioConfiguration;
+
+import java.io.File;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+/**
+ * Validates the Java native libs defined in the system property.
```

Comment:
Hadoop native lib may be defined here.

Reply:
But then the drawback is for the default value for your JVM, there's a good chance you don't have them all in your env and you always see this warning.......